google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITPaginationTest.java

```diff
@@ -114,13 +114,18 @@ public void testPaginationIterating() {
         ListZonesRequest.newBuilder().setProject(DEFAULT_PROJECT).setMaxResults(1).build();
     ZonesClient.ListPagedResponse response = zonesClient.list(listZonesRequest);
     boolean presented = false;
+    int count = 0;
```

Comment:
optional: Perhaps we could name this `numZones`?

Reply:
I go for `count` for shorter name.